amazon/ion/core.py

```diff
@@ -103,6 +104,18 @@ def is_stream_signal(self):
         return self < 0
 
 
+def scale_to_precision(fractional_seconds, fractional_precision):
```

Comment:
It's not. I changed `scale_to_precision` to use `Decimal.scaleb()`instead of the previous math formula I was using. I think using the `scale_to_precision` method to perform the scaleb action is beneficial, as it makes the code look cleaner.

Reply:
I am a bit ambivalent about this, so I think that is fine.